src/renderer/dx/CustomTextLayout.h

```diff
@@ -127,6 +127,7 @@ namespace Microsoft::Console::Render
 
         [[nodiscard]] HRESULT STDMETHODCALLTYPE _AnalyzeFontFallback(IDWriteTextAnalysisSource* const source, UINT32 textPosition, UINT32 textLength);
         [[nodiscard]] HRESULT STDMETHODCALLTYPE _SetMappedFont(UINT32 textPosition, UINT32 textLength, IDWriteFont* const font, FLOAT const scale);
+        [[nodiscard]] HRESULT STDMETHODCALLTYPE _SetMappedFontFace(UINT32 textPosition, UINT32 textLength, ::Microsoft::WRL::ComPtr<IDWriteFontFace5> const fontFace, FLOAT const scale);

```

Comment:
@miniksa The old `MapCharacters` gives us an `IDWriteFont`, which we pass into `_SetMappedFont`. The new `MapCharacters` however gives us an `IDWriteFontFace5` and so I had to create this function. Just wanted your eyes on it